README.md

```diff
@@ -100,6 +100,7 @@ npm install string-ts
   - [split](#split)
   - [toLowerCase](#tolowercase)
   - [toUpperCase](#touppercase)
+  - [startsWith](#startsWith)
```

Comment:
Could you keep alphabetical order here? I think it is the only place where it was overlooked.